superset/assets/javascripts/explore/components/controls/SelectControl.jsx

```diff
@@ -27,6 +31,9 @@ const defaultProps = {
   multi: false,
   onChange: () => {},
   showHeader: true,
+  optionRenderer: opt => opt.label,
+  valueRenderer: opt => opt.label,
```

Comment:
This is a bit confusing but essentially the `optionRenderer` applies to what's in the dropdown and the `valueRenderer` applies to the selected item(s). It's nice to have this level of control, for instance we could show the `expression` behind the metric only on the selected metric, not in the list if we wanted to.

So they both should show the label.